mshv-ioctls/src/ioctls/vm.rs

```diff
@@ -109,7 +109,7 @@ impl VmFd {
                 as_uint64: interrupt_type as u64 | ((control_flags as u64) << 32),
             },
             dest_addr: apic_id,
-            vector: vector,
+            vector: _vector,
```

Comment:
You can instead just write `vector` here because the field name and variable name are the same.